File: BEGINNER/Python/1013-TheGreatest.py
#recebe os três valores de uma vez só, sendo separados por um espaço
a, b, c = map(int, input().split())

#criar uma função que recebe dois valores verifica o maior
def MaiorNum(a, b):
    return (a+b+abs(a-b))/2

#verificar o maior entre a e b
MaiorAB = MaiorNum(a,b)
#verificar o maior entre o resultado de a e b com c
MaiorTotal = MaiorNum(MaiorAB, c)

#arredonda o valor do total e imprime
print(f"{round(MaiorTotal)} eh o maior")

File: BEGINNER/Java/1013-TheGreatest.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
         
        int a, b, c, maiorAB, maior;
         
        a = sc.nextInt();
        b = sc.nextInt();
        c = sc.nextInt();
        
        maiorAB = maiorNum(a, b);
        maior = maiorNum(maiorAB, c);
        
        System.out.println(maior + " eh o maior");
        
        sc.close();
    }
    
    public static int maiorNum(int x, int y) {
        return (x + y + Math.abs(x - y)) / 2;
    }
}
